yarn.lock

```diff
@@ -0,0 +1,2403 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'd say remove this file from git

Reply:
Unfortunately it appears it is recommended to have it here (for app development, not for libs) since it contains the exact node dependency versions in use. I had removed it and got really weird errors, since npm has some looser idea of version checks. That's what I've pieced together so far, at least, but I'm still quite new to the whole npm landscape.

I think it might be possible to make changes (to package.json?) that might allow us to get rid of yarn.lock without everything falling apart. I'll look into it.